README.md

```diff
@@ -1133,6 +1135,15 @@ A curated list of awesome Python frameworks, libraries and software. Inspired by
 * [Spyder](https://github.com/spyder-ide/spyder) - Open Source Python IDE.
 * [WingIDE](http://wingide.com/) - Commercial IDE for Python.
 
+# Services
+
+Online tools and APIs to simplify development.
+
+## Cloud Continuous Integration
+* [Travis CI.com](https://travis-ci.com) - Take care of running your tests and deploying your private apps.
+* [Travis CI.org](https://travis-ci.org) - A distributed build system for the open source community.
```

Comment:
@vinta I am not sure that CI section belongs here. It is not Python-specific and there are lots of such services. Maybe just link to https://github.com/pditommaso/awesome-ci ?